<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificationEvenementWindow</class>
 <widget class="QWidget" name="ModificationEvenementWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modification d'un événement</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="contactLabel">
         <property name="text">
          <string>Contact </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="contactComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="eventLabel">
         <property name="text">
          <string>Evénement </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="eventComboBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maxVisibleItems">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="modifEventLabel">
         <property name="text">
          <string>Modifier votre événement </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="modifEventLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="addTimeLabel">
         <property name="text">
          <string>Ajouter un délai à une tâche (en jours)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="addTimeSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="validButton">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitButton">
         <property name="text">
          <string>Quitter</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
